mathml/relations/css-styling/ignored-properties-001.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Ignored CSS properties</title>
+<link rel="help" href="https://mathml-refresh.github.io/mathml-core/#css-styling">
+<meta name="assert" content="Verify ignored style do not affect MathML layout.">
```

Comment:
Should this be reworded? Is it "Verify style with ignored properties does not affect MathML layout."?